region/maridia/outer.json

```diff
@@ -120,6 +120,117 @@
+                      "artificialMorph": false,
+                      "mode": "any"
+                    }},
+                    "f_MaridiaTubeBroken"
+                  ]
+                },
+                {
+                  "name": "Carry G-Mode Through Tube (Top to Left)",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [4],
+                      "artificialMorph": false,
+                      "mode": "any"
+                    }},
+                    "f_MaridiaTubeBroken",
+                    {"or":[
+                      "Gravity",
+                      "HiJump",
+                      "canSpringBallJumpMidAir"
+                    ]}
+                  ]
+                }
+              ]
+            },
+            {
+              "leavesWithArtificialMorph": true,
+              "strats": [
+                {
+                  "name": "Carry G-Mode Morph Through Tube (Right to Left)",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [3],
+                      "artificialMorph": true,
+                      "mode": "any"
+                    }},
+                    {"or":[
+                      "Morph",
+                      "SpringBall",
+                      {"and":[
+                        "Gravity",
+                        "h_canArtificialMorphIBJ"
+                      ]},
+                      {"and":[
+                        "Gravity",
+                        "h_canArtificialMorphBombHorizontally",
+                        {"or": [
+                          "Bombs",
+                          {"ammo": { "type": "PowerBomb", "count": 1 }}
+                        ]}
+                      ]},
+                      {"and":[
+                        { "not": "f_MaridiaTubeBroken" },
+                        "canRiskPermanentLossOfAccess"
+                      ]}
+                    ]}
+                  ]
+                },
+                {
+                  "name": "Carry G-Mode Morph Through Tube (Bottom to Left)",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2],
+                      "artificialMorph": false,
+                      "mode": "any"
+                    }},
+                    "f_MaridiaTubeBroken"
+                  ]
+                },
+                {
+                  "name": "Carry G-Mode Morph Through Tube (Top to Left)",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [4],
+                      "artificialMorph": false,
+                      "mode": "any"
+                    }},
+                    "f_MaridiaTubeBroken",
+                    {"or":[
```

Comment:
The requirements in `from[4]` and `from[2]` look swapped again.
Moving from 2 would need artificialmorph movement items too.